packages/hub/indexing/running-indexers.js

```diff
@@ -49,16 +51,43 @@ module.exports = class RunningIndexers {
     }
 
     let staticModels = flatten(this.staticModels);
-    let staticSchema = staticModels.filter(doc => this.schemaTypes.includes(doc.type));
-    this.sourcesUpdater.addStaticModels(staticSchema, staticModels);
+    let staticSchema = staticModels // TODO eventually we will not obtain schema from static-models
+      .filter(doc => this.schemaTypes.includes(doc.type))
+      .concat(this.cardSchemas);
+    this.sourcesUpdater.addStaticModels(staticSchema, staticModels.concat(this.cardSchemas));
+  }
+
+  get cardSchemas() {
+    if (this._cardSchemas) { return this._cardSchemas; }
+
+    let schemas = [];
+    let plugins = this.seedSchema.plugins;
+    let features = this.seedSchema.plugins.featuresOfType('schemas');
+    for (let feature of features) {
+      let { id:featureName } = feature;
+      let factory = plugins.lookupFeatureFactory('schemas', featureName);
+      // It's not clear to me how you could load card schema from different sources if we load card schema outside of a data-source context...
```

Comment:
You're correct that they all come from one source right now (the on-disk node_modules tree of cardstack plugins).

But this won't remain true for long. We plan to move away from having a static set of pre-installed cards.